polynote-frontend/polynote/ui/component/notebook/cell.ts

```diff
@@ -760,6 +760,8 @@ abstract class MonacoCell extends Cell {
         cellState.observeKey("editing", editing => {
             if (editing) {
                 this.editor.focus()
+            } else {
+                this.onBlur();
```

Comment:
This is a fix for a separate issue with raw markdown cells where unfocusing them sometimes does not render them - this was affecting the table of contents (where using it to navigate to a cell wasn't unfocusing the other one).